shared/teams/external-team.tsx

```diff
@@ -32,26 +37,256 @@ const ExternalTeam = (props: Props) => {
     )
   }, [getTeamInfo, teamname])
 
+  if (teamInfo) {
+    return (
+      <Kb.Box2 direction="vertical" gap="small" fullWidth={true} fullHeight={true}>
+        <ExternalTeamInfo info={teamInfo} />
+      </Kb.Box2>
+    )
+  }
+
   return (
     <Kb.Box2 direction="vertical" gap="small" style={styles.container}>
-      <Kb.Text type="Header">{teamname}</Kb.Text>
       {waiting ? (
         <Kb.ProgressIndicator />
```

Comment:
I think this progress indicator could be bigger, and maybe even get a "loading team..." or something - it's really small and if your connection is slow it looks like the app is borked.